prac_07/squaring.kv

```diff
@@ -15,11 +15,12 @@ BoxLayout:
         Label:
             id: output_label
             font_size: 48
+            color: (1,255,1,1)
 
     BoxLayout:
         orientation: 'vertical'
         multiline: False
         Label:
             text: ""
         Label:
-            text: 'Enter a number and press "Square"'
\ No newline at end of file
+            text: 'Enter a number and press "Square"'
```

Comment:
I'm pretty sure this Label requires an 'id' of what the Label is meant to be. It should also be indented to align with the 'vertical' BoxLayout at the top